package.json

```diff
@@ -1,84 +1,55 @@
 {
   "name": "@microbit/microbit-fs",
-  "version": "0.9.2",
+  "version": "0.10.0",
   "description": "Manipulate files in a micro:bit MicroPython Intel Hex file.",
   "keywords": [
     "microbit",
     "micropython",
     "filesystem",
     "fs"
   ],
-  "main": "./dist/bundles/microbit-fs.umd.js",
-  "mainMin": "./dist/bundles/microbit-fs.umd.min.js",
-  "module": "./dist/esm5/index.js",
-  "typings": "./dist/types/index.d.ts",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
+  "typings": "./dist/esm/index.d.ts",
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./dist/esm/index.d.js",
+        "default": "./dist/esm/index.js"
+      },
+      "require": {
+        "types": "./dist/cjs/index.d.js",
+        "default": "./dist/cjs/index.js"
+      }
+    }
+  },
   "sideEffects": false,
   "repository": {
     "type": "git",
     "url": "https://github.com/microbit-foundation/microbit-fs"
   },
-  "config": {
-    "umdName": "microbitFs"
-  },
   "author": "Micro:bit Educational Foundation <package-help@microbit.org>",
   "license": "MIT",
-  "engines": {
-    "node": ">=8.5",
-    "npm": ">=6.0",
-    "yarn": "^1.0"
-  },
   "scripts": {
-    "cleanup": "shx rm -rf dist",
-    "prebuild": "npm run cleanup && npm run verify",
-    "build": "tsc && rollup -c config/rollup.config.js",
-    "test": "jest -c ./config/jest.config.js",
-    "test:watch": "npm t -- --watch",
-    "test:coverage": "npm t -- --verbose --coverage",
-    "test:ci": "npm t -- --ci",
-    "validate-js": "tsc -p ./config",
-    "verify": "npm run validate-js && npm run style && npm run test:ci",
-    "style": "npm run format -- --list-different && npm run lint",
-    "style:fix": "npm run format:fix && npm run lint:fix",
-    "format": "prettier --config prettier.config.js \"**/*.{ts,tsx,js,jsx,css,scss,sass,less}\"",
-    "format:fix": "npm run format -- --write",
-    "lint": "tslint --project tsconfig.json --format codeFrame",
-    "lint:fix": "npm run lint -- --fix",
-    "docs": "typedoc --options config/typedoc.json"
-  },
-  "peerDependencies": {
-    "tslib": ">=1.9.0"
+    "build:esm": "tsc -p tsconfig.json && echo '{\"type\": \"module\"}' > dist/esm/package.json",
+    "build:cjs": "tsc -p tsconfig.cjs.json && echo '{\"type\": \"commonjs\"}' > dist/cjs/package.json",
+    "build": "npm run build:esm && npm run build:cjs",
+    "ci": "npm run build && npm run test && npx prettier --check .",
```

Comment:
No longer running lint checks in CI.

Reply:
Resolved by [2527ea2](https://github.com/microbit-foundation/microbit-fs/pull/46/commits/2527ea213d0abf9b1265d11faa67ede5ff011578)